prometheus/expvar_collector.go

```diff
@@ -59,6 +59,8 @@ type expvarCollector struct {
 // sample values.
 //
 // Anything that does not fit into the scheme above is silently ignored.
+//
+// Deprecated: Use collectors.NewExpvarCollector instead.
```

Comment:
Perhaps it's best to also remove the whole lengthy doc comment above, e.g.:

```go
// NewExpvarCollector is the obsolete version of collectors.NewExpvarCollector.
// See there for documentation.
//
// Deprecated: Use collectors.NewExpvarCollector instead.
```

The last line might seem redundant now, but it's important to have it in this format for linters.

Same for the doc comments for the other, now deprecated, functions.